public/src/componentHelper.tsx

```diff
@@ -371,11 +357,15 @@ export function getDaysAfterDeadline(deadline: Date, delivered: Date): number {
     return after > 0 ? after : 0;
 }
 
-export function isValidUserName(username: string): boolean {
-    const onlyCharsAndSpaces = /^[a-zA-Z\s]*$/;
-    return onlyCharsAndSpaces.test(username);
-}
 
 export function legalIndex(i: number, len: number): boolean {
     return i >= 0 && i <= len - 1;
 }
+
+export function forManualReview(a: Assignment): boolean {
```

Comment:
Use the same name as on the server-side, e.g., `GradedManually`.

Reply:
Done.